Python/Delete-Tweets/README.md

```diff
@@ -0,0 +1,50 @@
+
+# Delete Tweets Automatically using Python
+
+This is a utility to delete your tweets using python program
+
+## Requirements
+
+**credentials.json**: This file contains the credentials for accessing the twitter API. Generate the same from your twitter developer account and paste in this file
+
+***Format of 'credentials.json' file***
+
+    {
+        "consumer_key": "",
+        "consumer_key_secret": "",
+        "bearer_token": "",
+        "access_token": "",
+        "access_token_secret": ""
+    }
+
+### Usage
+
+After cloning the project in your local directory, download the required packages using the command:
+
+    pip3 install -r requirements.txt
+
+After all the required packages are installed, run the program using the following command.
+
+    python3 main.py --path P --param M [--count C] [--min N] [--max N] [--hours H] [--days D] [--verbose]
+
+#### Arguments
+
```

Comment:
I like the idea of the list, but i think it would look better as a code block.

```sh
--path P : Path to the 'credentials.json' file on your local system. It is a mandatory argument.

--param M : Method of filtering the tweets. Pass one of the following: ['retweet', 'likes', 'time']. It is also a mandatory argument.
```

Reply:
Done